### ISSUE TITLE
Lighthouse Feature Doesn't Work on Linode

### ISSUE THREAD

**Issue by dtemir** [MLH-Fellowship/gitcord-bot#69]
When trying to get the lighthouse stats from the bot deployed to Linode:

![Screenshot from 2021-08-11 20-41-20](https://user-images.githubusercontent.com/62047062/129125248-bffddba6-fe42-4422-bc42-7cae6102d179.png)

When running the bot from my local repo (up-to-date with `staging`, see second message)

![Screenshot from 2021-08-11 20-42-10](https://user-images.githubusercontent.com/62047062/129125315-71cdb9b2-0e87-4f42-9f68-013cc6f78ecf.png)

Maybe there's some issue with Linode not being able to use `chromeLauncher` due to some restrictions? (so that people wouldn't be able to DDoS from their servers)

**Comment by louisefindlay23** [comment]
It's to do with chromeLauncher I think. I couldn't get it work without a browser i.e. WSL2. Should look into it.

**Comment by Inoxia25** [comment]
@louisefindlay23 yup I also think so. Will search for an alternative to `chromelauncher`

**Comment by louisefindlay23** [comment]
@Inoxia25, merged your PR and tested it but there's a new error: `The CHROME_PATH environment variable must be set to a Chrome/Chromium executable no older than Chrome stable.',` `'ERR_LAUNCHER_PATH_NOT_SET'`
